homeassistant/components/volvo/icons.json

```diff
@@ -0,0 +1,92 @@
+{
+  "entity": {
+    "sensor": {
+      "availability": {
+        "default": "mdi:car-connected"
+      },
+      "average_energy_consumption": {
+        "default": "mdi:car-electric"
+      },
+      "average_energy_consumption_automatic": {
+        "default": "mdi:car-electric"
+      },
+      "average_energy_consumption_charge": {
+        "default": "mdi:car-electric"
+      },
+      "average_fuel_consumption": {
+        "default": "mdi:gas-station"
+      },
+      "average_fuel_consumption_automatic": {
+        "default": "mdi:gas-station"
+      },
+      "average_speed": {
+        "default": "mdi:speedometer"
+      },
+      "average_speed_automatic": {
+        "default": "mdi:speedometer"
+      },
+      "battery_capacity": {
+        "default": "mdi:car-battery"
+      },
+      "battery_charge_level": {
+        "default": "mdi:battery"
```

Comment:
This is probably already the default icon for `SensorDeviceClass.BATTERY`. Maybe this applies to others as well.